### PR TITLE
feat(web.domain): datagrid - display contact details

### PR DESCRIPTION
**Pull request by dectotam**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/domain-optim
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat MANAGER-11780
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

datagrid - display contact details

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/dashboard/domain.service.js [hunk 1]
```diff
@@ -91,6 +91,12 @@ angular.module('services').service(
       });
     }
 
+    getContactInfo(contactId) {
+      return this.OvhHttp.get(`/me/contact/${contactId}`, {
+        rootPath: 'apiv6',
+      });
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/dashboard/domain.service.js:97]:
```suggestion
      return this.$http.get(`/me/contact/${contactId}`).then(({data})=> data);
```

File: packages/manager/apps/web/client/app/domain/list/list-domain-layout.controller.js [hunk 2]
```diff
@@ -154,10 +166,18 @@ export default class ListDomainLayoutCtrl extends ListLayoutHelper.ListLayoutCtr
     });
   }
 
-  linkContactBuilder({ domain, whoisOwner }) {
+  getContactDetails(contactId, index) {
+    const self = this;
```
Comment (JacquesLarique) [inline packages/manager/apps/web/client/app/domain/list/list-domain-layout.controller.js:170]: I don't think you need to do this (webpack will take care of it)

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 3]
```diff
@@ -135,26 +135,101 @@
...
+    </oui-datagrid-column>
+
+    <oui-datagrid-column
+        data-title="::'domains_column_domains_technical_contact' | translate"
+        data-property="contactTech"
+        data-sortable
+    >
+        <a
+            class="oui-link_icon"
+            data-ng-if="$value.id === $ctrl.user.nichandle"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/list/template.html:218]:
```suggestion
            data-ng-if="$value.id === $ctrl.user.auth.account"
```
If the scope include CA. Same for all place where there are user.nichandle

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/list/template.html [hunk 4]
```diff
@@ -135,26 +135,101 @@
...
+                data-oui-popover-template="contactDetails.html"
+                data-oui-popover-open="$rowIndex === $ctrl.contactPopover.rowIndex"
+                data-ng-click="$ctrl.getContactDetails($value.id, $rowIndex)"
+            ></button>
+            <script type="text/ng-template" id="contactDetails.html">
+                <span>
+                    {{$ctrl.contactPopover.data.firstName}} {{$ctrl.contactPopover.data.lastName}}
+                    <br>
+                    {{$ctrl.contactPopover.data.organisationName}}
+                </span>
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/list/template.html:160]:
```suggestion
                <span data-ng-bind=":: $ctrl.contactPopover.data.firstName + ' ' + $ctrl.contactPopover.data.lastName">
                </span>
                <br />
                <span data-ng-bind=":: $ctrl.contactPopover.data.organisationName">
                </span>
```

**Pull request merged**